rootfs/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM quay.io/deis/base:0.3.0
+
+ENV VERSION=0.3.8.linux-amd64.go1.6.2
+# Add nsq user and group
+RUN adduser --system \
+	--shell /bin/bash \
+	--disabled-password \
+	--home /opt/nsq \
+	--group \
+	nsq
+
+RUN curl -Ls https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > /usr/bin/jq \
+    && chmod +x /usr/bin/jq \
+    && curl -s https://s3.amazonaws.com/bitly-downloads/nsq/nsq-$VERSION.tar.gz | tar xz \
+    && mv /nsq-$VERSION/bin/* /bin \
+    && rm -r /nsq-$VERSION
```

Comment:
Looks like spaces were used for indenting here with tabs having been used above in lines 6 - 10.  I won't start a Silicon Valley style tabs v spaces war, but within a single file, we should probably observe one convention or the other. If we wanted maximum consistency with our other Dockerfiles, we'd go with tabs.